src/mxdev/processing.py

```diff
@@ -205,7 +205,7 @@ def write_dev_sources(fio, packages: typing.Dict[str, typing.Dict[str, typing.An
             continue
         extras = f"[{package['extras']}]" if package["extras"] else ""
         subdir = f"/{package['subdirectory']}" if package["subdirectory"] else ""
-        install_options = " --pre"
+        install_options = ""
```

Comment:
empty install options? imo they can be removed entirely

Reply:
well, yes, why not